games-arcade/edgar/edgar-1.29.recipe

```diff
@@ -24,7 +24,7 @@ PROVIDES="
 REQUIRES="
 	haiku
 	lib:libintl
-	lib:libpng16
+	lib:libpng
```

Comment:
Why not libpng16?

Reply:
The `BUILD_REQURIES` already contains `devel:libpng16`. And since `libpng16` provides `libpng` with correct version constraints, using `lib:libpng` will allow us to just modify one occurance when we decide to bump `libpng` version

Reply:
IMO this is pointless to use tricks like this. It helps neither newcomers, nor consistency.

Reply:
The lib:libpng provide in the libpng package should be removed. The package indeed does not provide that library at all, only libpng16.

Reply:
If the general opinion is that it should be lib:libpng16 I'll bump it back again to use it like that